src/langchain_google_cloud_sql_mssql/mssql_engine.py

```
@@ -118,3 +118,33 @@ def connect(self) -> sqlalchemy.engine.Connection:
                 out from the connection pool.
         """
         return self.engine.connect()
+
+    def create_chat_history_table(self, table_name: str) -> None:
+        """Create table with schema required for MSSQLChatMessageHistory class.
+
+        Required schema is as follows:
+
+            CREATE TABLE {table_name} (
+                id INT IDENTITY(1,1) PRIMARY KEY,
+                session_id NVARCHAR(MAX) NOT NULL,
+                data NVARCHAR(MAX) NOT NULL,
+                type NVARCHAR(MAX) NOT NULL
+            )
+
+        Args:
+            table_name (str): Name of database table to create for storing chat
+                message history.
+        """
+        create_table_query = f"""IF NOT EXISTS (SELECT * FROM INFORMATION_SCHEMA.TABLES
+            WHERE TABLE_NAME = '{table_name}')
```

I left the `IF NOT EXISTS` here still as I feel like this is less error prone. Most likely these integrations will be run in scripts/notebooks or ran multiple times. We want methods to be able to be run more than once and not error out.

---

nit: I'm a little concerned by this because it seems like the table might exist but have the wrong schema. Can we throw an error if it's the wrong schema?

---

Discussed offline and if table already exists with malformed schema it will error with a nice error when user attempts to init `MSSQLChatMessageHistory`